share/sysbuild/image_configurations/MAIN_image_default.cmake

```diff
@@ -5,15 +5,16 @@
 # This sysbuild CMake file sets the sysbuild controlled settings as properties
 # on the main Zephyr image.
 
-set_config_bool(${ZCMAKE_APPLICATION} CONFIG_BOOTLOADER_MCUBOOT "${SB_CONFIG_BOOTLOADER_MCUBOOT}")
```

Comment:
signature and encryption parts moving is OK, mcuboot line moving is not and needs reverting

Reply:
Fixed